### PR TITLE
base code mismatch, Fix #15

### LINKED ISSUE #15: Code table mismatch
Hi all,

There is a mismatch between the code tables in the  [csv](https://github.com/multiformats/multibase/blob/master/multibase.csv) and the [readme](https://github.com/multiformats/multibase#multibase-table-v100-rc-semver).

More specifically we have :

| base | csv | readme |
| --- | --- | --- |
| base32 | Bb | Uu |
| base64 | m | y |
| base64url | u | Y |

Which one is authoritative ?

### PR DESCRIPTION
**Pull request by theobat**
Fix  [csv](https://github.com/multiformats/multibase/blob/master/multibase.csv) and [readme](https://github.com/multiformats/multibase#multibase-table-v100-rc-semver) mismatch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ base2         0       binary has 1 and 0
 base8         7       highest char in octal
 base10        9       highest char in decimal
 base16        F, f    highest char in hex
-base32        U, u    rfc4648 - highest letter
+base32        B, b    rfc4648 - highest letter
```
Comment (kevina) [inline README.md]: The comment highest letter is no longer accurate.

File: README.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ base2         0       binary has 1 and 0
...
 base32hex     V, v    rfc4648 - highest char
 base58flickr  Z       highest char
 base58btc     z       highest char
-base64        y       rfc4648 highest char
-base64url     Y       rfc4648 highest char
+base64        m       rfc4648 highest char
```
Comment (kevina) [inline README.md]: Ditto.  Also note we now have 'm' for base64 with no padding, and 'M' for the version with padding.  The 'M' stands for mime encoding.

**Comment by theobat** [comment]
@kevina, I'm fine with that, I could just remove the table in the readme and keep the link to the csv file. I think it's better not to duplicate information... (the current version of the readme is very confusing though so something should be done to improve the user experience)

**Comment by Kubuxu** [comment]
Resolved by: https://github.com/multiformats/multibase/pull/21

**Pull request closed without merging**